README.md

```diff
@@ -476,7 +476,7 @@ quickemu --vm windows-11-22H2.conf
 
 -   Complete the installation as you normally would.
 -   All relevant drivers and services should be installed automatically.
--   A local adminstrator user account is automatically created, with
+-   A local administrator user account is automatically created, with
```

Comment:
Corrections to README.md implemented in documentation sub-project so they will be present in regenerated docs.

Reply:
This will take care of all the changes in README.md and docs/*

Reply:
It has already been implemented in the https://github.com/philclifford/quickemu-docs/pull/12, so leaving as is. Guessing reverting is not necessary, but please correct me if I'm wrong.

Reply:
You're correct.